### PR TITLE
Change the request type to post fetch elements

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

File: src/test/java/org/gridsuite/study/server/SingleLineDiagramTest.java [hunk 1]
```diff
@@ -421,50 +428,50 @@ public void testDiagramsAndGraphics() throws Exception {
         assertTrue(TestUtils.getRequestsDone(1, server).contains(String.format("/v1/substations?networkUuid=%s&variantId=%s&substationId=BBE1AA&substationId=BBE2AA", NETWORK_UUID_STRING, VARIANT_ID)));
 
         //get the lines map data of a network
-        getNetworkElementsInfos(studyNameUserIdUuid, rootNodeUuid, "LINE", "MAP", List.of(), "[]");
-        getNetworkElementsInfos(studyNameUserIdUuid, rootNodeUuid, "LINE", "MAP", List.of("BBE1AA"), "[]");
+        getNetworkElementsInfos1(studyNameUserIdUuid, rootNodeUuid, "MAP", mapper.writeValueAsString(createRequestBody("LINE", List.of())), "[]");
```
Comment (jamal-khey) [inline src/test/java/org/gridsuite/study/server/SingleLineDiagramTest.java:431]: optional : use null for all mapper.writeValueAsString(createRequestBody("LINE", List.of())). 

but it' not bloking

**Review by jamal-khey — COMMENTED** [review]
use null instead of empty array to match the usage in the front end

File: src/test/java/org/gridsuite/study/server/SingleLineDiagramTest.java [hunk 2]
```diff
@@ -257,6 +257,13 @@ public MockResponse dispatch(RecordedRequest request) {
         server.setDispatcher(dispatcher);
     }
 
+    private Map<String, Object> createRequestBody(String elementType, List<String> substationsIds) {
+        Map<String, Object> requestBody = new HashMap<>();
+        requestBody.put("elementType", elementType);
+        requestBody.put("substationsIds", substationsIds);
+        return requestBody;
+    }
```
Comment (klesaulnier) [inline src/test/java/org/gridsuite/study/server/SingleLineDiagramTest.java:265]: make this method use "mapper.writeValueAsString" and return a string so you don't have to use "mapper.writeValueAsString" everywhere ?

**Review by klesaulnier — APPROVED** [review]
Test: OK
Code: OK

small test change

**Pull request merged**